### PR TITLE
[Stack Monitoring] Switch cgroup memory fields to keyword

### PR DESCRIPTION
**Pull request by matschaffer**
Rel https://github.com/elastic/beats/issues/31765

Per https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-nodes-stats.html they should be strings. The internal collection template (monitoring-es.json) has them as keywords without any `ignore_above`.

This template uses `ignore_above` on all keywords, so keeping that convention for these mapping updates.

Opening as draft since we won't want to merge this until there's a corresponding beats change in place to match.

### Testing

So far I can't figure out how to get ES to respond with `max` in the cgroup memory setting:

```
$ curl -s -k -u elastic:'(password)' https://localhost:9200/_nodes/stats/os | jq '.nodes | values | .[].os.cgroup.memory'
{
  "control_group": "/",
  "limit_in_bytes": "9223372036854771712",
  "usage_in_bytes": "17973219328"
}
```

I'll need that before I can advise on how to test this.

### CODE REVIEW

File: x-pack/plugin/core/src/main/resources/monitoring-es-mb.json [hunk 1]
```diff
@@ -732,14 +732,16 @@
                                 "usage": {
                                   "properties": {
                                     "bytes": {
-                                      "type": "long"
+                                      "ignore_above": 1024,
+                                      "type": "keyword"
```
[the remark this answers is not in the document]
Comment (matschaffer) [reply x-pack/plugin/core/src/main/resources/monitoring-es-mb.json:736]: Good question. I should at least check that we're not trying to graph it. ES returns a string (potentially `max`) so if we needed to keep it graphable I guess we could have metricbeat just drop the field if it's not a number.

[the remark this answers is not in the document]
Comment (matschaffer) [reply x-pack/plugin/core/src/main/resources/monitoring-es-mb.json:736]: I don't see it in https://github.com/elastic/kibana/blob/b8c1323d5bdc921a6238003bc698bc03750c5320/x-pack/plugins/monitoring/server/lib/metrics/elasticsearch/metrics.ts 

Additionally it's already `keyword` in internal collection so I think we're safe here. Would it be nice to aggregate cgroup memory, perhaps. But I'd probably opt to tackle that as a new feature.

**Review by miltonhultgren — APPROVED** [review]

**Comment by matschaffer** [comment]
Guessing I can probably merge this but maybe someone from @elastic/es-data-management should weigh in too. Also maybe they have ideas on how to get ES producing the `max` value seen in [forum posts](https://discuss.elastic.co/t/metricbeat-8-2-1-monitoring-elasticsearch-8-2-1-template-issue/305649/3).

**Review by jbaiera — APPROVED** [review]

**Comment by matschaffer** [comment]
Ah, so `9223372036854771712` _is_ max when running on docker for mac I'd guess. Interesting. Thanks @jbaiera !

**Comment by matschaffer** [comment]
Finally got the metricbeat suite to run against a docker image with this PR (`./gradlew buildDockerImage` + a follow-up PR to metricbeat mage setup). I'd say we're good to go.

**Pull request merged**